src/common/Input.component.tsx

```diff
@@ -0,0 +1,48 @@
+import { useField } from 'formik';
+import React from 'react'
+export interface Props {
+    formKey: string;
+    type: string;
+    placeholder: string;
+    label: string;
+    required: boolean;
+}
+
+const normalClassString = "bg-gray-50 \
```

Comment:
Doesn't make sense to have strings per component - we need to use theming on a higher level instead
https://tailwindcss.com/docs/theme